bundles/binding/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/EpsonProjectorBinding.java

```diff
@@ -522,7 +523,27 @@ private void sendDataToDevice(String deviceId, EpsonProjectorCommandType commman
 
         } catch (EpsonProjectorException e) {
             logger.warn("Couldn't execute command '{}', {}", commmandType, e.getMessage());
+            closeConnection(deviceId);
+        }
+    }
 
+    private void closeConnection(String deviceId) {
+        DeviceConfig device = deviceConfigCache.get(deviceId);
+
+        if (device == null) {
+            logger.error("Could not find device '{}'", deviceId);
```

Comment:
This should be a WARN instead of an ERROR.

Reply:
Done